### ISSUE TITLE
Molden Writer not writing [7F]

### ISSUE THREAD

**Issue by gayverjr** [psi4/psi4#2238]
Hi,
I am using the psi4/label/dev build from anaconda, and I noticed that the molden file produced by the following snippet does not write the [7F] line indicating spherical harmonic F-type basis functions to the molden file. [5D] and [9G] are there, so I'm assuming this isn't intentional.

```
import psi4

mol = psi4.geometry("""
    N 0.0000000000 0.0000000000 0.548756750
    N 0.0000000000 0.0000000000 -0.548756750
    """)

E, wfn = psi4.energy('scf/aug-cc-pvtz', return_wfn=True)
psi4.molden(wfn, 'n2.molden')
```
I'm not sure if this has been fixed elsewhere, but I didn't find it reported on here so I thought I would share. I have attached the molden file as well.

[n2.molden.txt](https://github.com/psi4/psi4/files/6773734/n2.molden.txt)

**Comment by JonathonMisiewicz** [comment]
Luckily, we're in the final stages of rewriting the Molden file writer anyways. Added this to our to-do list.

**Comment by JonathonMisiewicz** [comment]
Closed by #2229.
